docs/source/RegressionTestFarm.rst

```diff
@@ -126,9 +126,9 @@ Then I'd create each machine with the appropriate labels
     Added host:
         foo-amd.bazcorp.com
 
-    $ /usr/local/autotest/cli/autotest-rpc-client host create -t amd64 -b hostprovisioning foo-intel.bazcorp.com
+    $ /usr/local/autotest/cli/autotest-rpc-client host create -t intel64 -b hostprovisioning foo-intel.bazcorp.com
```

Comment:
@xutian  this line is add intel64 arch with foo-intel.bazcorp.com host, I did not delete amd64 adding host command line.